src/styles/StyleUtils.ts

```diff
@@ -471,12 +398,8 @@ function getBadgeColorStyle(success, error, isPressed = false, isAdHoc = false)
 
 /**
  * Generate a style for the background color of the button, based on its current state.
- *
- * @param {String} [buttonState] - One of {'default', 'hovered', 'pressed'}
- * @param {Boolean} isMenuItem - whether this button is apart of a list
- * @returns {Object}
  */
-function getButtonBackgroundColorStyle(buttonState = CONST.BUTTON_STATES.DEFAULT, isMenuItem = false) {
+function getButtonBackgroundColorStyle(buttonState: string = CONST.BUTTON_STATES.DEFAULT, isMenuItem = false): ViewStyle | CSSProperties {
```

Comment:
I think we can do something like this:
```suggestion
function getButtonBackgroundColorStyle(buttonState: ValueOf<typeof CONST.BUTTON_STATES> = CONST.BUTTON_STATES.DEFAULT, isMenuItem = false): ViewStyle | CSSProperties {
```

Reply:
Done